.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+.vscode
```

Comment:
@patrick-fs I actually intentionally checked in `.vscode` to ensure that people get the same formatting when editing. We do the same thing in our Sentry monolith.

Reply:
My bad!

Reply:
No worries! I can see why this looked like a mistake :)

Reply:
This has been reverted